File: angular/src/app/main/supplyChain/sales/shared/dtos/creditDebitNote-dto.ts
import { CreditDebitNoteDetailDto } from "./creditDebitNoteDetail-dto";

export class CreditDebitNoteDto{
	id:number|undefined;	
    tenantId:number|undefined;	
	locID :number|undefined;
	locDesc:string|undefined;
	docNo :number|undefined;
    docDate:any|undefined;
	postingDate :any|undefined;
	dateTime :any;
	paymentDate :any|undefined;
	typeID :number|undefined;
	accountID :string|undefined;
	accountName :string|undefined;
	subAccID :number|undefined;
	subAccName :string|undefined;
	reason :string|undefined;
	narration :string|undefined;
	totalQty :number|undefined;
	totAmt :number|undefined;
	transType:string|undefined;
	trTypeID:string|undefined;
	trTypeDesc:string|undefined;
    creditDebitNoteDetailDto:CreditDebitNoteDetailDto[] = new Array<CreditDebitNoteDetailDto>()
}

File: aspnet-core/src/ERP.Application.Shared/SupplyChain/Sales/Dtos/CreditDebitNoteDto.cs

using System;
using Abp.Application.Services.Dto;

namespace ERP.SupplyChain.Sales.Dtos
{
    public class CreditDebitNoteDto : EntityDto
    {
		public int LocID { get; set; }
        public string LocName { get; set; }

        public int DocNo { get; set; }

		public DateTime? DocDate { get; set; }

		public DateTime? PostingDate { get; set; }

		public DateTime? PaymentDate { get; set; }

		public short? TypeID { get; set; }

		public string AccountID { get; set; }

		public int? SubAccID { get; set; }

		public string Reason { get; set; }

		public string Narration { get; set; }

		public string OGP { get; set; }

		public double? TotalQty { get; set; }

		public double? TotAmt { get; set; }

		public bool Posted { get; set; }

		public int? LinkDetID { get; set; }

		public bool Active { get; set; }

		public string AudtUser { get; set; }

		public DateTime? AudtDate { get; set; }

		public string CreatedBy { get; set; }

		public DateTime? CreateDate { get; set; }
        public string TransType { get; set; }


    }
}